picachu/modules/galleries/queries/get_gallery_query.py

```diff
@@ -14,19 +14,9 @@ def by_id(cls, gallery_id: int) -> Optional[Gallery]:
         try:
             return current_session \
                 .query(Gallery) \
-                .get(gallery_id)
-        finally:
-            current_session.close()
-
-    @classmethod
-    def by_user_id(cls, current_user_id: int) -> List:
```

Comment:
Why is this method removed? It is needed in the endpoint to get all of the user's galleries.

Reply:
Such a function already exists. They are the same. That's why I removed one of them.